src/lwc/geFormField/geFormField.js

```diff
@@ -297,9 +309,13 @@ export default class GeFormField extends LightningElement {
         }
     }
 
+    get isRecordType() {
+        return this.fieldType === LOOKUP_TYPE && this.fieldApiName === RECORD_TYPE_FIELD.fieldApiName;
```

Comment:
Since we're also checking for the lookup type here and not just the record type fieldApiName, should we change the name of this getter to something like "isLookupRecordType" so that it's a little more clear? At first glance the name and the logic threw me some in terms of understand what was happening.